WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -277,8 +277,9 @@ private extension DashboardViewController {
         spacerView.isHidden = true
     }
 
-    func updateUI(site: Site?) {
-        guard let siteName = site?.name, siteName.isEmpty == false else {
+    func updateUI(site: Site) {
+        let siteName = site.name
+        guard siteName.isEmpty == false else {
```

Comment:
Super nit: We could also use the extension property `isNotEmpty` for simplicity here.
```suggestion
        guard siteName.isNotEmpty else {
```

Reply:
This looks nicer! Updated in https://github.com/woocommerce/woocommerce-ios/pull/6864/commits/edf5ac27f368a72e0b010cd890917269434f3f72